assets/launcher/lang/sklmessages_de_DE.properties

```diff
@@ -66,91 +66,91 @@ alert.button.cancel=Abbrechen
 
 # About
 about.translation=Übersetzt von:
-about.translation.authors=X0R3
+about.translation.authors=X0R3, mistermodcreator
 
 # Login Screen
-login.button.type.microsoft=Login with Microsoft
-login.button.type.offline=Login offline
-login.label.username=Username
-login.dropdown.name=Logged In Users
-login.button.play=Play
-login.button.logout=Log Out
-login.button.switch.offline=Switch to offline mode
-login.button.switch.online=Switch to online mode
+login.button.type.microsoft=Mit Microsoft-Konto anmelden
+login.button.type.offline=Offline anmelden
+login.label.username=Nutzername
+login.dropdown.name=Angemeldete Nutzer
+login.button.play=Spielen
+login.button.logout=Abmelden
+login.button.switch.offline=In den Offline-Modus wechseln
+login.button.switch.online=In den Online-Modus wechseln
 
 # Settings
-settings.header.general=General
+settings.header.general=Allgemein
 settings.header.updates=Updates
-settings.header.about=About
-settings.footer.button.save=Save
-
-settings.option.language.name=Language
-settings.option.language.info=You need to restart the launcher to apply the changes.
-settings.option.accent.name=Accent Color
-settings.option.theme.name=Theme
-settings.option.theme.dark.name=Dark Theme
-settings.option.sorting.name=Profiles Sorting
-settings.option.sorting.bylastplayed.name=Sort profiles by last played
-settings.option.sorting.byname.name=Sort profiles by name
-settings.option.default.name=Default Profiles
-settings.option.default.hide.name=Hide Default Profiles
-settings.option.console.name=Launcher Console
-settings.option.console.show.name=Show Launcher Console
-settings.about.contributors=Thanks to all contributors:
+settings.header.about=Über SKlauncher
```

Comment:
There is. But I thought that it would be clearer what is meant if the name of the launcher is in there. Also, I've seen a translation like this in other software. And besides, the german `Über`, which is the translation of `about`, can also mean above, which could confuse people.